apps/condo/domains/common/components/TelephonyContext.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useCallback, useEffect } from 'react'
+
+import { usePostMessageContext } from './PostMessageProvider'
+import { RequestHandler } from './PostMessageProvider/types'
+
+export const TelephonyContext = React.createContext({})
+
+export const TelephonyContextProvider = ({ children }) => {
+    const { addEventHandler } = usePostMessageContext()
+    
+    const handleMessage = useCallback<RequestHandler<'CondoWebAppCall'>>(async ({ phone }) => {
+        if (typeof window === 'undefined') return null
+
+        window.open(`/phone/+${phone}`, '_blank')
+    }, [])
+
+    useEffect(() => {
+        addEventHandler('CondoWebAppCall', '*', handleMessage)
+    }, [addEventHandler, handleMessage])
+
+    return (
+        <TelephonyContext.Provider value={{}}>
```

Comment:
Why is context needed here?

Reply:
Rename it to `TelephonyEventsHandler`, remove context